### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by nathan7432**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName='Login' screenOptions={{
+        headerShown: false
+      }}>
+        <Stack.Screen name='Login' component={LoginScreen} />
+        <Stack.Screen name='CreateP' component={CreatePScreen} />
+        <Stack.Screen name='Main' component={MainScreen} />
+        <Stack.Screen name='Friends' component={FriendsScreen} />
+        <Stack.Screen name='CreateM' component={CreateMScreen} />
+        <Stack.Screen name='Profile' component={ProfileScreen} />
```
Comment (ascala7) [inline App.js:23]: move files to src folder

File: CreateMScreen.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { StatusBar } from 'expo-status-bar';
+import { StyleSheet, Text, View, Button } from 'react-native';
+
+const CreateMScreen = ( {navigation} ) => {
```
Comment (ascala7) [inline CreateMScreen.js]: add jsdocs to screens

**Comment by ascala7** [comment]
delete Startup.jsx file

**Review by ascala7 — CHANGES_REQUESTED** [review]

File: CreateMScreen.js (hunk 2 above)
Comment (nathan7432) [reply CreateMScreen.js]: added

**Pull request merged**